### PR TITLE
[Cognitive Services / Language] Dynamic classification `class` to `classifications`

### PR DESCRIPTION
**Pull request by omaremadeldin**
# Data Plane API - Pull Request
Is this review for (select one):

- [x] a private preview
- [ ] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>

Renaming the property that contains the classification results for Dynamic Classification from `class` to `classifications`, this is a bug in the swagger, the actual served content has always been classifications, so no breaking change there (SDKs have not released yet and they need this change to be able to release).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json [hunk 1]
```diff
@@ -752,7 +752,7 @@
     "ClassificationDocumentResult": {
       "type": "object",
       "properties": {
-        "class": {
+        "classifications": {
```
Comment (weidongxu-microsoft) [inline specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json:755]: I don't think you can change property name already in public preview.

Comment (omaremadeldin) [reply specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json:755]: this is a bug in the swagger, the actual served content has always been classifications, so no breaking change there (SDKs have not released yet and they need this change to be able to release), plus this feature is gated private preview and we haven't allowed any customers yet to use it.

Comment (weidongxu-microsoft) [reply specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json:755]: Ok. I think I get the context. I will let Heath or others to sign off.

Meanwhile, check errors in ModelValidation

**Review by mssfang — APPROVED** [review]

**Review by kristapratico — CHANGES_REQUESTED** [review]

**Review by deyaaeldeen — APPROVED** [review]
Looks good. I tried it and the tests pass without any hacks/swagger transforms.

**Review by kristapratico — COMMENTED** [review]
Thank you! @heaths are you able to help merge? This matches the current service behavior in 2022-10-01-preview.

**Review by kristapratico — APPROVED** [review]

**Comment by kristapratico** [comment]
@omaremadeldin it looks like the x-ms-example files for DynamicClassification need to be updated to match the change we're making in this PR

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
@akning-ms @lmazuel  how can we suppress these constant LRO issues? They started recently, and even if they are old, none of the LRO authoring has changed to cause this.

Suppressing.

**Comment by heaths** [comment]
Also, this is a correction to pass the service responses.

**Pull request merged**